src/main/java/org/graylog/integrations/dataadapters/GreyNoiseQuickIPDataAdapter.java

```diff
@@ -92,11 +93,15 @@ protected void doRefresh(LookupCachePurge cachePurge) throws Exception {
 
     @Override
     protected LookupResult doGet(Object keyObject) {
+        String userAgent = String.format("Graylog/%d.%d",
+                Version.CURRENT_CLASSPATH.getVersion().getMajorVersion(),
+                Version.CURRENT_CLASSPATH.getVersion().getMinorVersion());
```

Comment:
I think just using the string representation of the version object would return the full version, which would be a bit simpler. Thinking of the server startup message example:

https://github.com/Graylog2/graylog2-server/blob/10c3095e40e30f65f2c5936a968a98b05fc69e7a/graylog2-server/src/main/java/org/graylog2/bootstrap/ServerBootstrap.java#L128

> Graylog server 4.3.0-SNAPSHOT starting up